charts/hono/Chart.yaml

```diff
@@ -15,11 +15,14 @@ apiVersion: v1
 version: 1.0.2
 # Version of the application being deployed by the chart
 appVersion: 1.0.2
-name: eclipse-hono
+name: hono
 description: Scalable IoT messaging platform
 keywords:
   - IoT
   - messaging
 home: https://www.eclipse.org/hono/
 sources:
   - https://github.com/eclipse/hono
+maintainers:
+  - name: dejanb
+    email: dbosanac@redhat.com
```

Comment:
running `helm lint` locally on the 1.0.2 chart succeeds on my machine, the only thing it recommends is to add an icon. Did you get a different result when running `helm lint`?

Reply:
Yes, thus the changes: However the CI pipeline doesn't call "helm lint" directly, but runs a GitHub action: https://github.com/eclipse/packages/blob/b1905b68999ece4e1242ae687fff8de836a0f26d/.github/workflows/ci.yaml#L38-L48